src/PrestaShopBundle/Translation/PrestaShopTranslatorTrait.php

```diff
@@ -209,4 +203,23 @@ private function normalizeDomain($domain)
 
         return $normalizedDomain;
     }
+
+    /**
+     * Returns the message
+     *
+     * @param string $message the message
+     * @param string $functionName Name of function to be called
```

Comment:
```suggestion
     * @param string|callable $functionName Name of function to be called
```

Reply:
Only this one to fix and we should be good